pkgs/by-name/po/poutine/package.nix

```diff
@@ -15,13 +16,22 @@ buildGoModule rec {
     hash = "sha256-T81Qi79VaZzfKL4niTZQW+gwwiBcyInALrvyUg1V4Ck=";
   };
 
+  nativeBuildInputs = [ installShellFiles ];
+
   vendorHash = "sha256-/chq40j+puAI3KdI15vbZwrnzCKrU7g+Z/t9aOXQ1Sg=";
 
   ldflags = [
     "-s"
     "-w"
   ];
 
+  postInstall = ''
```

Comment:
Could you add `lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform)` to improve cross compilation?